tests/src/test/scala/org/dhallj/tests/acceptance/HashingPreludeSuites.scala

```diff
@@ -1,66 +1,66 @@
 package org.dhallj.tests.acceptance
 
-class HashingPreludeBoolAndSuite extends HashingSuite("semantic-hash/success/prelude/Bool/and")
-class HashingPreludeBoolBuildSuite extends HashingSuite("semantic-hash/success/prelude/Bool/build")
-class HashingPreludeBoolEvenSuite extends HashingSuite("semantic-hash/success/prelude/Bool/even")
-class HashingPreludeBoolFoldSuite extends HashingSuite("semantic-hash/success/prelude/Bool/fold")
-class HashingPreludeBoolNotSuite extends HashingSuite("semantic-hash/success/prelude/Bool/not")
-class HashingPreludeBoolOddSuite extends HashingSuite("semantic-hash/success/prelude/Bool/odd")
-class HashingPreludeBoolOrSuite extends HashingSuite("semantic-hash/success/prelude/Bool/or")
-class HashingPreludeBoolShowSuite extends HashingSuite("semantic-hash/success/prelude/Bool/show")
+class HashingBoolAndSuite extends HashingSuite("semantic-hash/success/prelude/Bool/and")
```

Comment:
I think I'd like to keep it easy to distinguish the prelude tests from others (both in the results and with `testOnly`). I'm not necessarily against removing the `Prelude` from the test class name, but if we do I think these should go in a new `org.dhallj.tests.acceptance.prelude` package.

Reply:
Oh sorry!!! That was an unintentional `sed` change. Will revert